importer/src/main/java/org/hiero/mirror/importer/migration/AsyncJavaMigration.java

```diff
@@ -63,21 +65,33 @@ with last as (
             where f.installed_rank = last.installed_rank
             """;
 
-    protected final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+    private final ObjectProvider<JdbcTemplate> jdbcTemplateProvider;
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
```

Comment:
We can just store `NamedParameterJdbcOperations` and use `namedParameterJdbcOperations.getJdbcOperations()` when needed.

```suggestion
    private final ObjectProvider<NamedParameterJdbcOperation> namedParameterJdbcOperation;
```